packages/langium/src/dependency-injection.ts

```diff
@@ -74,12 +74,20 @@ export function eagerLoad<T>(item: T): T {
 function _inject<I, T>(module: Module<I, T>, injector?: any): T {
     const proxy: any = new Proxy({} as any, {
         deleteProperty: () => false,
-        get: (obj, prop) => _resolve(obj, prop, module, injector || proxy),
+        set: () => {
+            throw new Error('Cannot set property on injected service container');
+        },
+        get: (obj, prop) => {
+            if (prop === isProxy) {
+                return true;
+            } else {
+                return _resolve(obj, prop, module, injector || proxy);
+            }
+        },
         getOwnPropertyDescriptor: (obj, prop) => (_resolve(obj, prop, module, injector || proxy), Object.getOwnPropertyDescriptor(obj, prop)), // used by for..in
         has: (_, prop) => prop in module, // used by ..in..
-        ownKeys: () => [...Reflect.ownKeys(module), isProxy] // used by for..in
+        ownKeys: () => [...Object.getOwnPropertyNames(module)] // used by for..in
```

Comment:
What is the difference here? The documentation says it is the same, except that symbols are not listed with `Object.getOwnPropertyNames`.
What is the idea behind this change? On first sight it looks more complicated, because you need to handle `isProxy` more.

Reply:
`isProxy` is a very special variable that should not be exposed from the service container using reflection. We only need it internally.